moduleroot/.github/workflows/modulesync.yml.erb

```
@@ -0,0 +1,51 @@
+---
+# Managed by modulesync - DO NOT EDIT
+# https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
+
+name: CI
+
+on: pull_request
+
+jobs:
+  compare_with_repo:
+    name: 'Setup Test Matrix'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Puppet module repository
+        uses: actions/checkout@v2
+        path: modules/${{ env.github_repository }}
+
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.0'
+
+      - name: Set modulesync version
+        run: cat .msync.yml | ruby -ryaml -e 'puts "MSYNC_VER=#{YAML.load(STDIN)["modulesync_config_version"]}"' >> $GITHUB_ENV
```

what about this:
```suggestion
        run: ruby -ryaml -e 'puts "MSYNC_VER=#{YAML.safe_load(File.read(".msync.yml"))["modulesync_config_version"]}"' >> $GITHUB_ENV
```

```console
$ ruby -ryaml -e 'puts "MSYNC_VER=#{YAML.safe_load(File.read(".msync.yml"))["modulesync_config_version"]}"'
MSYNC_VER=4.1.0
```

Allows us to spare one pipe